hydra-cluster/hydra-cluster.cabal

```diff
@@ -133,6 +133,8 @@ library
     , directory
     , filepath
     , hspec-expectations
+    , http-conduit
+    , http-types
```

Comment:
Are we really using both to match the `HttpException`?

Reply:
I removed both and used http-client instead since it contains all we need.